gitgud/skills/user_messages.py

```diff
@@ -179,46 +186,25 @@ def display_entry(index, human_name, code_name, indent):
             )
 
 
-def target_branch_str():
-    file_operator = operations.get_operator()
-    tree = file_operator.get_current_tree()
-    referred_by = {}
-    for branch_name in tree['branches']:
-        target = tree['branches'][branch_name]['target']
-        if target not in referred_by:
-            referred_by.update({target: [branch_name]})
-        else:
-            referred_by[target].append(branch_name)
-    for target in referred_by:
-        referred_by[target] = ", ".join(referred_by[target])
-    return referred_by
-
-
-def display_tree_data(header, data, show_content, file_count=2):
+def display_tree_content(header, content, show_content=True, file_count=2):
     file_format_str = "  {path} - {content}"
 
-    def existence_str(condition):
-        if condition:
-            return "Exists"
-        else:
-            return "Doesn't exist"
-
-    data_paths = list(data.keys())
+    available_files = list(content.keys())
     # If there are more files than we requested, we want to show that too
 
     print(header + ":")
-    for index in range(max(file_count, len(data_paths))):
-        file_exists = index < len(data_paths)
-        # Handle number of tracked files
-        if file_exists:
-            path = data_paths[index]
-        else:
-            path = "File " + str(index + 1)
-        if show_content and file_exists:
-            content = data[data_paths[index]]
-        else:
-            content = existence_str(file_exists)
-        print(file_format_str.format(path=path, content=content))
+    for filepath in available_files:
+        file_content = filepath if show_content else existence_str(True)
```

Comment:
Mistake here, it should be `file_content = content[filepath] ...`

Also, the name `file_content` is a bit misrepresentative since it doesn't show the content. Maybe `file_details`/`file_info` or something more encapsulating? If you do that, can you similarly change the name in the format string also